.travis.yml

```diff
@@ -33,6 +33,10 @@ cache:
 before_cache:
   - rm -f $HOME/.cache/pip/log/debug.log
 
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y unrar
+
```

Comment:
I strongly believe that we should disable the provider, but I'll need to check it later.
With the current implementation, as soon as you get a rar file an exception will be raised and no subtitle will be downloaded. If we decide to not disable the provider then I'll need to change the logic a little bit in order to proceed with the other possible candidates.